usr/src/man/man3nvpair/nvlist_add_boolean.3nvpair

```diff
@@ -251,6 +251,10 @@ thread can actively change a given \fBnvlist_t\fR at a time. The caller is
 responsible for the synchronization.
 .sp
 .LP
+The name used for any name-value pair must be less than 2^15 bytes in
+length, otherwise EINVAL will be returned.
```

Comment:
I think it's pretty close to what we want. I think I'd include something about the NUL terminator being included. I might make the allusion to INT16_MAX just for clarify, but not sure if that's better or not. So maybe something like:

```
The name used for any name-value pair, including the terminating null byte, can be no more than
.B INT16_MAX
(2^15 - 1) bytes in
length, otherwise
.B EINVAL
will be returned.
```

Don't forget to run git pbchk which will run mandoc's lint on this.